G/Gnuplot/build_tarballs.jl

```diff
@@ -0,0 +1,64 @@
+
+using BinaryBuilder, Pkg
+
+name = "Gnuplot"
+version = v"5.4.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://downloads.sourceforge.net/project/gnuplot/gnuplot/$(version)/gnuplot-$(version).tar.gz",
+                  "6b690485567eaeb938c26936e5e0681cf70c856d273cc2c45fabf64d8bc6590e"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/gnuplot-*/
+
+# Don't try this at home, it's evil
+ln -s /opt/${host_target}/${host_target}/sys-root/usr/lib/libc.so /usr/lib/libc.so
+
+if [[ "${target}" == "${MACHTYPE}" ]]; then
+    # Delete system libexpat to avoid confusion
+    rm /usr/lib/libexpat.so*
+elif [[ "${target}" == *-mingw* ]]; then
+    # Apply patch from https://github.com/msys2/MINGW-packages/blob/5dcff9fd637714972b113c6d3fbf6db17e9b707a/mingw-w64-gnuplot/01-gnuplot.patch
+    atomic_patch -p1 ../patches/01-gnuplot.patch
+    autoreconf -fiv
+fi
+
+export CPPFLAGS="$(pkg-config --cflags glib-2.0) $(pkg-config --cflags cairo) $(pkg-config --cflags pango) -I$(realpath term)"
+export LDFLAGS="-liconv"
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} 
+cd src
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("gnuplot", :gnuplot),
+    #ExecutableProduct("gnuplot_qt", :gnuplot_qt, "$libexecdir")
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    BuildDependency(PackageSpec(name="Xorg_xorgproto_jll", uuid = "c4d99508-4286-5418-9131-c86396af500b")),
+    #Dependency(PackageSpec(name="Libcerf_jll", uuid="af83a40a-c4c4-57a0-81df-2309fbd279e3")),
+    Dependency(PackageSpec(name="LibGD_jll", uuid="16339573-6216-525a-b38f-30b6f6b71b5f")),
+    Dependency(PackageSpec(name="Lua", uuid="a4086b1d-a96a-5d6b-8e4f-2030e6f25ba6")),
+    Dependency(PackageSpec(name="Qt5Tools_jll", uuid="a9c6e4b1-b2fb-56d5-96a9-25f276f13840")),
+    Dependency(PackageSpec(name="Qt5Svg_jll", uuid="3af4ccab-a251-578e-a514-ea85a0ba79ee")),
+    Dependency(PackageSpec(name="Qt5Base_jll", uuid="ea2cea3b-5b76-57ae-a6ef-0a8af62496e1")),
```

Comment:
Do we need all of these, especially at runtime?  The [archlinux package](https://archlinux.org/packages/extra/x86_64/gnuplot/) doesn't use `qt5-base`, and `qt5-tools` is a build dependency only

Reply:
Correct, tools should be a build-dep, and Qt5Svg already pulls in Qt5Base.